logstash-core/spec/logstash/instrument/wrapped_write_client_spec.rb

```diff
@@ -97,7 +112,7 @@
   end
 
   context "AckedMemoryQueue" do
-    let(:queue) { LogStash::Util::WrappedAckedQueue.create_memory_based("", 1024, 10, 1024) }
+    let(:queue) { LogStash::Util::WrappedAckedQueue.create_memory_based("", 1024, 10, 4096) }
```

Comment:
See https://github.com/elastic/logstash/issues/7031#issuecomment-301283267 for reasoning.